app/model/scoreboard.js

```diff
@@ -0,0 +1,135 @@
+import Target from './target.js';
+import Round from './round.js';
+
+
+/**
+ * Keeps track of the scoreboard and it's affiliated stats
+ */
+export default class Scoreboard {
+
+    constructor() {
+        let targetTypes = ['20', '18', '19', '17', '16', '15', 'B'];
+        this._targets = [];
+        for(let targetType of targetTypes) {
+            this._targets.push({type: targetType, counter: new Target()});
+        }
+        this._activeTargetIndex = 0;
+        this._activeTarget = this._targets[this._activeTargetIndex];
+
+        this._round = new Round();
+    }
+
+    /**
+     * Marks a dart that hits. Moves to next target as appropriate.
+     * Ignored if scoreboard is already done.
+     * @returns {Scoreboard}
+     */
+    hit() {
+        if(!this.isDone()) {
```

Comment:
hmm. yeah right now it just shows the final scorecard. i think when i add a 'new game' or something like that it will be more relevant